sql_patch.php

```diff
@@ -33,7 +33,7 @@
 <head>
 <title>LibreHealth EHR <?php echo attr($EMRversion) ?> <?php echo xlt('Database Patch'); ?></title>
 <link rel='STYLESHEET' href='interface/themes/style_setup.css'>
-<link rel="shortcut icon" href="public/images/favicon.ico" />
+<link rel="shortcut icon" href="favicon.ico" />
```

Comment:
Should these be under assets/images?

Reply:
favicon.ico is usually held in a webroot for the browser tab to pick up automatically.  It could be anywhere though.